source/plugin/Assets/GoogleMobileAds/Platforms/Unity/BannerClient

```diff
@@ -0,0 +1,238 @@
+        };
+        
+        public BannerClient()
+        {
+            prefabAd = null;
+            dummyAd = null;
+        }
+
+        // Creates a banner view and adds it to the view hierarchy.
+        public void CreateBannerView(string adUnitId, AdSize adSize, AdPosition position)
+        {
+            Debug.Log("Dummy " + MethodBase.GetCurrentMethod().Name);
+            prefabAd = Resources.Load(prefabAds[adSize]) as GameObject;
+
+            if (adSize == AdSize.SmartBanner)
+            {
+                AdBehaviour.SetAndStretchAd(prefabAd, position);
+
+            }
+            else
+            {
+                AdBehaviour.AnchorAd(prefabAd, position);
+            }
+        }
+    
+        // Creates a banner view and adds it to the view hierarchy with a custom position.
+        public void CreateBannerView(string adUnitId, AdSize adSize, int x, int y)
+        {
+            Debug.Log("Dummy " + MethodBase.GetCurrentMethod().Name);
+            
+            prefabAd = Resources.Load(prefabAds[adSize]) as GameObject;
+            Image myImage = prefabAd.GetComponentInChildren<Image>();
+            RectTransform rect = myImage.GetComponent<RectTransform>();
+
+            if (adSize == AdSize.SmartBanner)
+            {
+                AdBehaviour.SetAndStretchAd(prefabAd, 0);
+                rect.anchoredPosition = new Vector3(0, y, 1);
+            }
+            else
+            {
+                rect.anchoredPosition = new Vector3(x, y, 1);
+            }
+            
+        }
+
+        // Requests a new ad for the banner view.
+        public void LoadAd(AdRequest request)
+        {
+            Debug.Log("Dummy " + MethodBase.GetCurrentMethod().Name);
+            ShowBannerView();
+            
+        }
+
+        // Shows the banner view on the screen.
+        public void ShowBannerView()
+        {
+            Debug.Log("Dummy " + MethodBase.GetCurrentMethod().Name);
+            
+            Image myImage = prefabAd.GetComponentInChildren<Image>();
+            RectTransform rect = myImage.GetComponent<RectTransform>();
+            dummyAd = AdBehaviour.ShowAd(prefabAd, rect.anchoredPosition);
+            
+        }
+
+        // Hides the banner view from the screen.
+        public void HideBannerView()
+        {
+            Debug.Log("Dummy " + MethodBase.GetCurrentMethod().Name);
+            
+            AdBehaviour.DestroyAd(dummyAd);
+        }
+
+        // Destroys a banner view.
+        public void DestroyBannerView()
+        {
+            Debug.Log("Dummy " + MethodBase.GetCurrentMethod().Name);
+            if (dummyAd != null)
+            {
+                AdBehaviour.DestroyAd(dummyAd);
+            }
+        }
+
+        // Returns the height of the BannerView in pixels.
+        public float GetHeightInPixels()
+        {
+            Debug.Log("Dummy " + MethodBase.GetCurrentMethod().Name);
+            return 0;
```

Comment:
this is fine, but if it's easy enough to know the width and height of the prefab (if loaded) that might be cool to return.